tui/src/ui/model/mod.rs

```diff
@@ -430,6 +431,8 @@ impl Model {
         );
 
         let media_root = Self::loading_tree();
+        Self::discover_servers(tx_to_main.clone()).await;
```

Comment:
I would recommend to put this as a tokio task instead of forcing to wait for DLNA before showing the TUI.

Reply:
This is strange... when I use ```tokio::spawn```, I get "future cannot be sent between threads". When I use it exactly as in the songtext with ```tokio::task::block_in_place```, then it does exactly what it says: it blocks., so the starting is delayed, the ui is shown later.